compiler/wrap.js

```diff
@@ -147,6 +147,39 @@ export default (source, flags = [ 'module' ], customImports = {}, print = str =>
   times.push(performance.now() - t1);
   if (Prefs.profileCompiler) console.log(bold(`compiled in ${times[0].toFixed(2)}ms`));
 
+  const printDecomp = (middleIndex, func, funcs, globals, exceptions) => {
+    console.log(`\x1B[35m\x1B[1mporffor backtrace\u001b[0m`);
+    console.log('\x1B[4m' + func.name + '\x1B[0m');
+    
+    const surrounding = Prefs.debugSurrounding ?? 5;
+    let min = middleIndex - surrounding;
+    let max = middleIndex + surrounding + 1;
+    if (Prefs.debugVerbose) {
```

Comment:
Can we also just print the entire function when `middleIndex` is ` -1`? Also could you rename this pref to something more ~obvious like `--backtrace-entire-func` or something similar? ty

Reply:
Done :)